### PR TITLE
Adding postgreSQL cluster and monitoring details

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>

**What this PR does / why we need it**:
    This PR is adding PostgreSQL cluster, monitoring and permission manifest.
      
      - PostsgreSQL
      - Zalando - pg operator 
      - Master as (weak)
      - Master and Slave as (Resilient)
      - Jmeter load generator
      - Custom load generator
      - Prometheus
      - Grafana

### CODE REVIEW

File: custom/workflow-helper/postgres-helper/application/deploy-load.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      app: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      serviceAccountName: postgres-operator # require permission is get resource "secrets" in the namespace lirmus 
+      containers:
+      - name: postgres
+        image: kaleoum/postgres:ld
```
Comment (ispeakc0de) [inline custom/workflow-helper/postgres-helper/application/deploy-load.yaml:21]: we can update this

File: custom/workflow-helper/postgres-helper/application/permission-&-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,316 @@
+apiVersion: v1
+kind: ConfigMap
```
Comment (ispeakc0de) [inline custom/workflow-helper/postgres-helper/application/permission-&-configmap.yaml:2]: we can rename the file -- avoid & symbol

File: custom/workflow-helper/postgres-helper/application/permission-&-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+subjects:
+- kind: ServiceAccount
+  name: postgres-operator
+  namespace: postgres
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: postgres-pod
```
Comment (ispeakc0de) [inline custom/workflow-helper/postgres-helper/application/permission-&-configmap.yaml:282]: I didn't see any ClusterRoleBinding for this ClusterRole

Comment (oumkale) [reply custom/workflow-helper/postgres-helper/application/permission-&-configmap.yaml:282]: I am referring the [service account](https://github.com/zalando/postgres-operator/blob/master/manifests/operator-service-account-rbac.yaml).

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**